sources/packages/web/src/views/aest/institution/InstitutionNotes.vue

```diff
@@ -1,55 +1,57 @@
 <template>
-  <v-card class="mt-4">
-    <div class="mx-5 py-4">
-      <v-row class="mb-2 mt-2">
-        <v-col cols="3" class="category-header-large">Notes</v-col>
-        <v-col class="text-center">
-          <div class="float-right">
-            <v-row>
-              <v-col>
-                <v-btn
-                  rounded
-                  variant="tonal"
-                  :class="{ 'primary-btn-background': !filteredNoteType }"
-                  data-cy="allNotesButton"
-                  @click="filterNotes()"
-                  >All Notes</v-btn
-                >
-              </v-col>
-              <v-col v-for="item in InstitutionNoteType" :key="item">
-                <v-btn
-                  rounded
-                  variant="tonal"
-                  :class="{
-                    'primary-btn-background': filteredNoteType === item,
-                  }"
-                  data-cy="noteTypeItem"
-                  @click="filterNotes(item)"
-                  >{{ item }}</v-btn
-                >
-              </v-col>
-            </v-row>
-          </div>
-        </v-col>
-      </v-row>
-      <content-group>
-        <Notes
-          title="Past Notes"
-          :entityType="NoteEntityType.Institution"
-          :notes="notes"
-          @submitData="addNote"
-        ></Notes>
-      </content-group>
-    </div>
-  </v-card>
+  <v-container>
+    <v-row class="my-2 pl-0">
+      <v-col cols="3" class="category-header-large">Notes</v-col>
+      <v-col class="text-center">
+        <div class="float-right">
+          <v-row>
+            <v-col>
+              <v-btn
+                rounded
+                :color="!filteredNoteType ? 'primary' : 'secondary'"
+                data-cy="allNotesButton"
+                @click="filterNotes()"
+                >All Notes</v-btn
+              >
+            </v-col>
+            <v-col v-for="item in InstitutionNoteType" :key="item">
+              <v-btn
+                rounded
+                :color="filteredNoteType === item ? 'primary' : 'secondary'"
+                data-cy="noteTypeItem"
+                @click="filterNotes(item)"
+                >{{ item }}</v-btn
+              >
+            </v-col>
+          </v-row>
+        </div>
+      </v-col>
+    </v-row>
+    <full-page-container
```

Comment:
Not part of the PR but this page is not following the layout of the other ones, right? Probably a page from before the introduction of the components.